index.js

```diff
@@ -188,7 +188,15 @@ function formatLicense(license) {
 
 module.exports = function checkPath(packageName, basePath, overrides, includeDevDependencies, includeOptDependencies) {
     if (!fs.existsSync(basePath)) {
-        return null
+        var dirs = basePath.split(path.sep)
+        if (dirs.length > 2) {
+            // Instead, search for the module in the parent node_modules folder
+            var moduleDirs = dirs.slice(dirs.length - 2, dirs.length)
+            var baseDirs = dirs.slice(0, dirs.length - 4)
+            return checkPath(packageName, baseDirs.concat(moduleDirs).join(path.sep), overrides);
```

Comment:
I think they are functionally equivalent.

`basePath` is the result of a call to `path.join`, so it will definitely be separated by `path.sep` characters. `path.sep` is an illegal filename character on both Windows and Linux, so `split` should never split in the middle of a filename.

I did originally write the join code as `path.join.apply(path, baseDir.concat(moduleDirs))`, but I thought it was ugly so I changed it.